contracts/ForkDelta.sol

```diff
@@ -66,6 +91,15 @@ contract ForkDelta {
   function changeFreeUntilDate(uint freeUntilDate_) public isAdmin {
     freeUntilDate = freeUntilDate_;
   }
+  
+  /// Updates the contract by setting a successor.
+  function updateContract(address successor_) public isAdmin {
```

Comment:
Can we name this in a more describing way? Like updateSuccessor. 

And just out of habit add a: require(successor != address(0)); ?